lib/endpoints/class-wp-json-posts-controller.php

```diff
@@ -9,6 +9,76 @@ public function __construct( $post_type ) {
 	}
 
 	/**
+	 * Register the routes for the objects of the controller
+	 * 
+	 */
+	public function register_routes() {
+
+		parent::register_routes();
```

Comment:
Let's remove this unless it's needed. The base controller shouldn't need to register endpoints, right?